src/main/ml-modules/root/server/endpoints/adhoc/api-adhoc-search.xqy

```diff
@@ -124,8 +124,12 @@ declare function local:get-result()
     else
       ()
 
-  return
-    search-lib:search($searchParams,$database,($export-csv eq "true"))
+  let $ret := search-lib:search($searchParams,$database,($export-csv eq "true"))
+  let $_ := if ( $cfg:D ) then
+              xdmp:log(("Returned from search-lib",$ret))
```

Comment:
How about a "data-explorer" trace event? I generally favor no xdmp:log statements in a codebase, but lots of xdmp:trace.